### ISSUE TITLE
Create a PR template

### ISSUE THREAD

**Issue by rgbkrk** [nteract/archived-desktop-app#1080]
Since we have some of our commit guidance buried away in a guide, we should [create a PR template that GitHub can pick up](https://github.com/blog/2111-issue-and-pull-request-templates). I'm guessing we could make it similar to redux-observable's:

```
<!-- If this is your first PR for nteract, please mark these boxes to confirm (otherwise you can exclude them)-->

- [ ] I have read the [Contributor Guide](https://github.com/nteract/nteract/blob/master/CONTRIBUTING.md)
- [ ] My commit messages are in [conventional-changelog-standard](https://github.com/nteract/nteract/blob/master/CONTRIBUTING.md#how-should-i-write-my-commit-messages-and-pr-titles) format.
```

Open to opinions here, might as well start with something.

**Comment by jdetle** [comment]
Thought: We could include `Questions? Feel free to ping us on [slack](https://slack.nteract.in)`

**Comment by LBKinson** [comment]
I was unsure of PR title verbage, so I would encourage your second bullet to be:
`- [ ] My PR title and commit messages are in [conventional-changelog-standard](https://github.com/nteract/nteract/blob/master/CONTRIBUTING.md#how-should-i-write-my-commit-messages-and-pr-titles) format.`

**Comment by rgbkrk** [comment]
> I was unsure of PR title verbage... My PR title and commit messages...

Whoa! Totally. Glad you noticed.

**Comment by lgeiger** [comment]
This was closed in #1082 
Thanks to @vperezma 👍
